cmd/syselevenexporter/syselevenexporter.go

```diff
@@ -61,27 +63,43 @@ var rootCmd = &cobra.Command{
 		log.Infof(version.Info())
 		log.Infof(version.BuildContext())
 
-		if os.Getenv("OS_USERNAME") == "" {
-			log.Fatalf("OS_USERNAME is missing")
+		if os.Getenv("OS_APPLICATION_CREDENTIAL_ID") == "" || os.Getenv("OS_APPLICATION_CREDENTIAL_SECRET") == "" {
+			useAppCreds = false
+		} else {
+			useAppCreds = true
 		}
 
-		if os.Getenv("OS_PASSWORD") == "" {
-			log.Fatalf("OS_PASSWORD is missing")
-		}
+		if !(useAppCreds) {
```

Comment:
I'd move those checks up in the `if` clause above. I'd also add a check wether `OS_PROJECT_ID` is defined when using application credentials :)